cli/module_registry.go

```diff
@@ -637,3 +645,35 @@ func getEntrypointForVersion(mod *apppb.Module, version string) (string, error)
 	// if there is no entrypoint set yet, use the last uploaded entrypoint
 	return mod.Entrypoint, nil
 }
+
+func isTarball(path string) bool {
+	return strings.HasSuffix(path, ".tar.gz") || strings.HasSuffix(path, ".tgz")
+}
```

Comment:
This is a very weak check. I am on the fence as to whether we should make it a better test using a tar + gzip reader but that would require an extra scan. Opinions?